oauthdoctor/diag/config_test.go

```diff
@@ -252,6 +255,52 @@ func TestPrint(t *testing.T) {
 	}
 }
 
+func TestReplaceConfig(t *testing.T) {
+	log.SetOutput(ioutil.Discard)
+
+	now := time.Now().Format("2006-01-02_")
+	dir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Error getting current dir: %s", err)
+	}
+
+	test := struct {
+		desc   string
+		cfg    ConfigFile
+		backup string
+	}{
+		desc: "Config file and backup file exist",
+		cfg: ConfigFile{
+			Filepath: filepath.Join(dir, "testdata"),
+			Filename: "python_config2",
+			Lang:     "python",
+		},
+		backup: "diag/testdata/python_config2_" + now,
+	}
+
+	backup := test.cfg.ReplaceConfig(DevToken, "randomToken")
+	config := test.cfg.GetFilepath()
+	_, err = os.Stat(config)
+	if os.IsNotExist(err) {
+		t.Fatalf("%s\nConfig file (%s) is missing", test.desc, config)
+	} else if err != nil {
+		t.Fatalf("%s\nProblem finding the config file: %s", test.desc, config)
```

Comment:
If you are testing for err != nil you should expose err in Fatalf so we know the exact error. It should be specific like.

"Cannot find config file %s: %s", config, err